ports/ticoban/Ticoban.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/ticoban/
```

Comment:
```suggestion
GAMEDIR=/$directory/ports/ticoban
```
GAMEDIR is wrong =) needs to be GAMEDIR=/$directory/ports/ticoban see the / which results in 
`no such file: '/roms/ports/ticoban//ticoban/ticoban.pyxapp'`